app/scripts/modules/amazon/pipeline/stages/cloneServerGroup/awsCloneServerGroupStage.js

```diff
@@ -93,5 +95,30 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.aws.cloneServerGr
     this.processIsSuspended = (process) => {
       return stage.suspendedProcesses && stage.suspendedProcesses.includes(process);
     };
+
+    this.blockDeviceMappingsSource = () => {
```

Comment:
Should prefix the name with a verb to indicate what it does, e.g. `getBlockDeviceMappingSource`